src/open_inwoner/openzaak/clients.py

```diff
@@ -95,18 +105,17 @@ def fetch_cases_by_bsn(
         user_bsn: str,
         max_requests: int | None = 4,
         identificatie: str | None = None,
+        max_confidentiality: VertrouwelijkheidsAanduidingen = VertrouwelijkheidsAanduidingen.openbaar,
```

Comment:
Passing `max_confidentiality` to `fetch_cases` is obligatory, so the default value here should be unnecessary.